playbooks/installation/installation.adoc

```diff
@@ -22,21 +23,53 @@ If adding those entries is problematic, a quick DNS solution for OpenShift is to
 
 === Node Sizing
 
-The Offcial Documentation lists some link:https://docs.openshift.com/enterprise/3.0/admin_guide/install/prerequisites.html#system-requirements[minimal system requirements] for OpenShift masters and nodes. Beyond that, there are several things to consider when deciding on the sizing for your nodes.
```

Comment:
Shouldn't we link to the minimum system requirements instead of duplicating them?